packages/shared/src/utils/url/url.ts

```diff
@@ -159,6 +159,19 @@ export const getStaticUrl = (path = '', is_document = false, is_eu_url = false)
     return `${host}${lang}/${normalizePath(path)}`;
 };
 
+export const getHubSignupUrl = () => {
+    const current_domain = process.env.NODE_ENV === 'production' ? deriv_urls.HUB_PRODUCTION : deriv_urls.HUB_STAGING;
+    let lang = default_language?.toLowerCase();
+
+    if (lang && lang !== 'en') {
+        lang = `?lang=${lang}`;
+    } else {
```

Comment:
Do we need this else block, can handle the english in the first if block?

Reply:
Aiteee will change the code 🙏